Tests/SonarQube.Bootstrapper.Tests/BootstrapperExeTests.cs

```diff
@@ -200,6 +201,30 @@ public void Exe_PreProc_VersionCheckSucceeds_PreProcSucceeds()
         }
 
         [TestMethod]
+        public void Exe_PostProc_ExecutableNotFound_PostProcFails()
+        {
+            // Arrange
+            string rootDir = TestUtils.CreateTestSpecificFolder(this.TestContext);
+            using (InitializeNonTeamBuildEnvironment(rootDir))
+            {
```

Comment:
Please add a comment saying that you are purposely not creating a DummyPostProcessor like the other tests do.